### PR TITLE
Dark Theme Bug Fixed

### PR DESCRIPTION
**Pull request by adityadigant**

### CODE REVIEW

**Comment by adityadigant** [comment]
The reference Code for Unexpected Token Commit: https://web.dev/offline-cookbook/

self.addEventListener('fetch', function (event) {
  event.respondWith(
    // Try the cache
    caches
      .match(event.request)
      .then(function (response) {
        // Fall back to network
        return response || fetch(event.request);
      })
      .catch(function () {
        // If both fail, show a generic fallback:
        return caches.match('/offline.html');
        // However, in reality you'd have many different
        // fallbacks, depending on URL and headers.
        // Eg, a fallback silhouette image for avatars.
      }),
  );
});

**Comment by shikhar0507** [comment]
> The reference Code for Unexpected Token Commit: https://web.dev/offline-cookbook/
> 
> self.addEventListener('fetch', function (event) {
> event.respondWith(
> // Try the cache
> caches
> .match(event.request)
> .then(function (response) {
> // Fall back to network
> return response || fetch(event.request);
> })
> .catch(function () {
> // If both fail, show a generic fallback:
> return caches.match('/offline.html');
> // However, in reality you'd have many different
> // fallbacks, depending on URL and headers.
> // Eg, a fallback silhouette image for avatars.
> }),
> );
> });

?

File: src/sw.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ self.addEventListener('fetch', (event) => {
             // console.log('Network request for ', event.request.url);
             return fetch(event.request).then(fetchResponse => {
                 if (fetchResponse.status === 404) {
-                    return caches.match('error-404.html');
+                    return caches.match('/error-404.html');
```
Comment (shikhar0507) [inline src/sw.js:87]: the name `error-404` is what's put in cache. see the `const files =[...]` array values

**Comment by adityadigant** [comment]
changed this "caches.match('offline.html')" and "caches.match('error-404.html');" with "caches.match('/offline.html')" and "caches.match('/error-404.html');" respectively.

File: src/sw.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ self.addEventListener('fetch', (event) => {
 
         }).catch(error => {
             console.log(error)
-            return caches.match('offline.html')
+            return caches.match('/offline.html')
```
Comment (shikhar0507) [inline src/sw.js:102]: This is already working as expected. Issue is not because of this

**Pull request merged**